README.md

```diff
@@ -32,33 +32,51 @@ The bot can assign and remove specified roles to a user temporarily.
+MoonlightBot uses Discord's built-in permissions system to control who is and is not be able to execute certain commands. Some commands have required permissions set by default, and overrides for specific users and roles can be applied to all commands. To set up permissions properly, please follow the [Permission Tutorial](./start-up/permission-tutorial.md).
 
 ## Logging
 
-MoonlightBot offers configurable logging, and can log several kinds of actions to one or more channels. To enable and configure logging for a specific channel, use the command
+MoonlightBot offers highly-configurable logging, and can log several kinds of actions to one or more channels. To enable and configure logging for a specific channel, use the command
 ```
 /config channels channel:LOG-CHANNEL logs:Open editor
 ```
 where `LOG-CHANNEL` is the channel you want logs posted to.
 
-It will ask you for the log actions, and you can enter your choice of items or categories from the [list of log names](./advanced/list-of-log-names.md) or simply enter an asterisk (`*`) to log everything. The list of items and categories to log should be separated by commas and spaces, like so: `BAN, KICK, USER_JOIN, USER_LEFT`
+An editor will open where you can enter items or categories from the [list of log names](./advanced/list-of-log-names.md), or an asterisk (`*`) to log everything. The list of items and categories to log should be separated by commas and spaces, like so: `BAN, KICK, USER_JOIN, USER_LEFT`
+
+![Log editor popup](./.gitbook/assets/LogEditor.png)
 
 ## Mute Setup
 
-MoonlightBot can mute users both temporarily and permanently. To use mute features, use the command
+MoonlightBot can mute users both temporarily and permanently. Use the command
 ```
 /create-muterole
 ```
-to automatically create a mute role with the correct permissions. You can specify a role to be used, or leave it blank and the bot will automatically create a new role.
+to set up a mute role. Specifying the `role` [option](./start-up/options.md) allows you to set up an existing role, or you can leave it out to create a new `@Muted` role.
 
 ![Result of /create-muterole command](./.gitbook/assets/MainPageMuterole.png)
 
-You should now be able to use [`/mute`](./moderation-commands/mute.md),  [`/tempmute`](./moderation-commands/tempmute.md), and [`/unmute`](./moderation-commands/unmute.md).
+You will now be able to use [`/mute`](./moderation-commands/mute.md),  [`/tempmute`](./moderation-commands/tempmute.md), and [`/unmute`](./moderation-commands/unmute.md).
+
+## Evasion Bans
+
+Evasions bans are a fallback moderation feature to ensure muted users cannot abuse improperly configured channels. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command
```

Comment:
```suggestion
Evasions bans are a fallback moderation feature to ensure muted users cannot abuse improperly configured permissions. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command
```

So what happens is that setting an override to allow it will take priority over deny overrides, this aims at falling back or escalating a mute to a harsher ban.

This also requires a Premium sub, you can insert the vote free trial here!